### ISSUE TITLE
Error when trying to delegate to a validator on ZetaChain

### ISSUE THREAD

**Issue by fadeev** [leapwallet/cosmos-extension#35]
<img width="396" alt="Screenshot 2023-09-20 at 13 19 11" src="https://github.com/leapwallet/cosmos-extension/assets/332151/058cf840-484c-4b4f-8462-b006ade014f4">

May be related to these queries to the `txs` endpoint.

<img width="1473" alt="Screenshot 2023-09-20 at 13 26 35" src="https://github.com/leapwallet/cosmos-extension/assets/332151/5d6ee362-7fc7-4429-9983-cfb059d9459a">

<img width="1473" alt="Screenshot 2023-09-20 at 13 26 37" src="https://github.com/leapwallet/cosmos-extension/assets/332151/6980fc82-a5f5-441e-8c87-e7d5e17c9ef4">

Claiming staking rewards also fails.

<img width="396" alt="Screenshot 2023-09-20 at 13 24 23" src="https://github.com/leapwallet/cosmos-extension/assets/332151/1e65b28a-97df-4fd1-9e45-250890fd4de1">

**Comment by Kneva** [comment]
The same error with the Keplr wallet.

**Comment by Kneva** [comment]
![image](https://github.com/leapwallet/cosmos-extension/assets/83719898/0dde8f7e-2595-4baf-baf5-b27a08ad3ccf)  Now another error occurs.

**Comment by baryon2** [comment]
@fadeev @Kneva thank you for reporting this issue. Can you share the chain info for ZetaChain that you used to add to Leap?

```
   rpc: string;
   rest: string;
   chainId: string;
   chainName: string;

   stakeCurrency: Currency;
   walletUrlForStaking?: string;
   bip44: {
        coinType: number;
    };
   bech32Config: {
      bech32PrefixAccAddr: string
   };
   currencies: AppCurrency[];
   feeCurrencies: FeeCurrency[];
```

**Comment by Kneva** [comment]
![image](https://github.com/leapwallet/cosmos-extension/assets/83719898/b43ff095-0279-4f8b-b899-de58ca143fd4)
https://developers.leapwallet.io/chain-store  Added a chain through the Chain Store

**Comment by himanshu-leap** [comment]
Hey, thanks @Kneva 
We will check what's happening here and will fix it.

**Comment by Kneva** [comment]
@himanshu-leap I hope you will succeed.

**Comment by hsnice16** [comment]
Thanks for pointing this out. 
The issue is happening because all the nodes are down. I don't think we can do much here, but still, we are looking into it. I'll keep here posted.

**Comment by himanshu-leap** [comment]
Thanks for pointing this out.
The issue is happening because all the nodes are down. I don't think we can do much here, but still, we are looking into it. I'll keep here posted.

**Comment by himanshu-leap** [comment]
Yeah, but, are you able to do txs using curl?
It throws 400 status, but nothing in the response.

Also, Is this updated?
<img width="1405" alt="Screenshot 2023-10-11 at 1 41 16 PM" src="https://github.com/leapwallet/cosmos-extension/assets/110231645/35d56527-0c70-420d-9f6f-ce69ae4407e3">

**Comment by fadeev** [comment]
@himanshu-leap good catch! Submitted a PR: https://github.com/cosmos/chain-registry/pull/2952

**Comment by fadeev** [comment]
Updated:

<img width="1840" alt="Screenshot 2023-10-12 at 11 47 15" src="https://github.com/leapwallet/cosmos-extension/assets/332151/83d0318d-3167-49ae-8c39-95bc81963062">

**Comment by himanshu-leap** [comment]
Great! 

Can you check this?
> Are you able to do the transaction using curl?
It throws 400 status, but nothing in the response. What's the exact error?

**Comment by fadeev** [comment]
@himanshu-leap I don't exactly remember how to post txs via `curl`, but here's sending a bank send tx using the same RPC endpoint on the CLI:

```
zetacored tx bank send alice zeta1k0z6vt8xv7s3nzqakqeqm7v893qscrw7mfg5mf 1azeta --node https://zetachain-athens.blockpi.network:443/rpc/v1/public --chain-id athens_7001-1
```

```
auth_info:
  fee:
    amount: []
    gas_limit: "200000"
    granter: ""
    payer: ""
  signer_infos: []
  tip: null
body:
  extension_options: []
  memo: ""
  messages:
  - '@type': /cosmos.bank.v1beta1.MsgSend
    amount:
    - amount: "1"
      denom: azeta
    from_address: zeta19nfaqu9wr0fktyyampva98ec025kjy0phww5um
    to_address: zeta1k0z6vt8xv7s3nzqakqeqm7v893qscrw7mfg5mf
  non_critical_extension_options: []
  timeout_height: "0"
signatures: []
confirm transaction before signing and broadcasting [y/N]: y
code: 0
codespace: ""
data: ""
events: []
gas_used: "0"
gas_wanted: "0"
height: "0"
info: ""
logs: []
raw_log: '[]'
timestamp: ""
tx: null
txhash: 0298C49F267B7116666366DAADDDA8D8926CFB5604F91E55FC5449D5DB26F0CE
```
